SwrveConversationSDK/Conversation/SwrveConversationAtom.h

```diff
@@ -3,6 +3,16 @@
 
 @class SwrveBaseConversation;
 
+#define kSwrveKeyTag @"tag"
+#define kSwrveKeyType @"type"
+#define kSwrveKeyValues @"values"
+#define kSwrveKeyAnswerId @"answer_id"
+#define kSwrveKeyAnswerText @"answer_text"
+#define kSwrveKeyStyle @"style"
+#define kSwrveKeyFontFile @"font_file"
+#define kSwrveKeyTextSize @"text_size"
+#define kSwrveKeyAlignment @"alignment"
+
```

Comment:
~~ These could fit the other constant patterns since there's technically no need for a macro here. it's just holding an NSString, which could be a const instead.